libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -405,6 +405,8 @@ namespace eosio { namespace testing {
             cfg.eosvmoc_config.cache_size = 1024*1024*8;
 
             for(int i = 0; i < boost::unit_test::framework::master_test_suite().argc; ++i) {
+               // don't use auto tier up for tests, since the point is to test diff vms
+               cfg.eosvmoc_tierup = chain::wasm_interface::vm_oc_enable::oc_none;
```

Comment:
Not sure why this is inside the `for()` loop instead of just with the `cfg.` items a few lines above.